Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_DeepSpeech_From_Tensorflow.md

[Pretrained English speech-to-text model](https://github.com/mozilla/DeepSpeech/releases/tag/v0.8.2) is publicly available.
To download the model, follow the instruction below:

* For UNIX-like systems, run the following command:
```
wget -O - https://github.com/mozilla/DeepSpeech/archive/v0.8.2.tar.gz | tar xvfz -
wget -O - https://github.com/mozilla/DeepSpeech/releases/download/v0.8.2/deepspeech-0.8.2-checkpoint.tar.gz | tar xvfz -
```
* For Windows systems:

>   1. Download the archive with the model: [https://github.com/mozilla/DeepSpeech/archive/v0.8.2.tar.gz](https://github.com/mozilla/DeepSpeech/archive/v0.8.2.tar.gz).

Reviewer: ```suggestion
  1. Download [the archive with the model](https://github.com/mozilla/DeepSpeech/archive/v0.8.2.tar.gz).
```

>   2. Download the TensorFlow MetaGraph with pretrained weights: [https://github.com/mozilla/DeepSpeech/releases/download/v0.8.2/deepspeech-0.8.2-checkpoint.tar.gz](https://github.com/mozilla/DeepSpeech/releases/download/v0.8.2/deepspeech-0.8.2-checkpoint.tar.gz).

Reviewer: ```suggestion
  2. Download the [TensorFlow MetaGraph with pretrained weights](https://github.com/mozilla/DeepSpeech/releases/download/v0.8.2/deepspeech-0.8.2-checkpoint.tar.gz).
```

After that, you will get the pretrained frozen model file `output_graph.pb` in the directory `deepspeech` created at
the beginning. The model contains the preprocessing and main parts. The first preprocessing part performs conversion of input
spectrogram into a form useful for speech recognition (mel). This part of the model is not convertible into
the IR because it contains unsupported operations `AudioSpectrogram` and `Mfcc`.

The main and most computationally expensive part of the model converts the preprocessed audio into text.
There are two specificities with the supported part of the model.

The first is that the model contains an input with sequence length. So the model can be converted with

> a fixed input length shape, thus the model is not reshape-able.

Reviewer 2: ```suggestion
a fixed input length shape, thus the model is not reshapable.
```

Refer to the [Using Shape Inference](../../../../OV_Runtime_UG/ShapeInference.md) guide.

The second is that the frozen model still has two variables: `previous_state_c` and `previous_state_h`, figure
with the frozen *.pb model is below. It means that the model keeps training these variables at each inference.

![DeepSpeech model view](../../../img/DeepSpeech-0.8.2.png)

> At the first inference the variables are initialized with zero tensors. After executing, the results of the `BlockLSTM*

Reviewer 2: ```suggestion
At the first inference, the variables are initialized with zero tensors. After execution, the results of the `BlockLSTM'
```

are assigned to cell state and hidden state, which are these two variables.

> ## Converting the Main Part of DeepSpeech Model into the IR

Reviewer 2: ```suggestion
## Converting the Main Part of DeepSpeech Model into OpenVINO IR
```
